Makefile

```diff
@@ -4,7 +4,7 @@ all: help
 
 boots: cmd/boots/boots ## Compile boots for host OS and Architecture
 
-ipxe: build_all_ipxe ## Build all iPXE binaries 
```

Comment:
it doesn't make sense to be a verb form now that its a variable instead of a target

Reply:
sorry, I'm not familiar with this way of naming make targets. is this format of naming documented somewhere?

Reply:
build_all_ipxe is a verb as it implies action. But this isn't a target any more its a list of files, so we'd want a noun right?

There's no standard/naming convention on this, its just sort of how the Makefile is written (but not 100% either).

It reads weird to have `build_all_ipxe := file1 file2`. Also its no longer `build_all_ipxe` but `generated_ipxe_files` so that its consistent with `generated_go_files`.

Reply:
Don't agree with this, but it's definitely not doing any harm so won't hold up the PR for it.